frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -1614,13 +1607,7 @@ const ManageHostsPage = ({
     }
 
     // There are no hosts for this instance yet
-    if (
-      !status &&
```

Comment:
we need this or a status filter wont render if there are 0 hosts with that status